docs/initial.md

```diff
@@ -0,0 +1,31 @@
+# OCFL Community Extension Initial
+
+  * **Extension Name:** initial
+  * **Authors:** N Jefferies
+  * **Minimum OCFL Version:** 1.0
+  * **OCFL Community Extensions Version:** 1.0
+  * **Obsoletes:** n/a
+  * **Obsoleted by:** n/a
+
+*Note: This is a placeholder for the special extension name "initial".*
+
+## Overview
+
+This extension definition ensures that the special extension name _initial_ is valid registered extension name.
+
+In practice, this extension MUST be replaced by a functional extension as detailed below. 
+
+## Parameters
+
+None
+
+## Optional _initial_ Extension 
+
+A _root extension directory_ MAY optionally contain an _initial_ extension that, if it exists, SHOULD be applied before all other extensions in the directory.
```

Comment:
Based on the discussion in the [comments of the PR](https://github.com/OCFL/extensions/pull/74#pullrequestreview-1911796064), the body of this extension should likely be rewritten.